io.github.ezQuake.yaml

```diff
@@ -0,0 +1,112 @@
+app-id: io.github.ezQuake
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "19.08"
+command: ezquake.sh
+
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --share=ipc
+- --share=network
+- --socket=pulseaudio
+- --persist=.ezquake
+
+cleanup:
+- /app/include
+- /app/lib/*.a
+- /app/lib/*.la
+- /app/lib/pkgconfig
+
+modules:
+
+- name: jansson
+  buildsystem: cmake-ninja
+  sources:
+  - type: archive
+    url: http://www.digip.org/jansson/releases/jansson-2.12.tar.gz
+    sha256: 5f8dec765048efac5d919aded51b26a32a05397ea207aa769ff6b53c7027d2c9
+
+- name: ezquake-source
+  buildsystem: simple
+  build-commands:
+  - make
+  - install -Dm 744 ezquake-linux-x86_64 /app/bin/ezquake-linux-x86_64
+  sources:
+  - type: archive
+    url: https://github.com/ezQuake/ezquake-source/archive/3.1.tar.gz
+    sha256: 58a02594f0d4a632c3be308abb18b05cded85a23e28e6b2a5fc3f49cd4dcd24e
+
+# We must do something really fishy. ezQuake has no support for mutiple users
+# or anything in that direction. Just download the binary and copy paste it next
+# to your existing Quake binary. As such, there is no difference between user resources
+# and static app resources. To fix this in the context of a Flatpak, we create a series 
+# of static resources in the /app/share folder and we create a copy for the user to edit
+# and change. To bind it all together, we use symlinks.
+
+# More modern games do exactly this, but they have it as part of the application. Here
+# We add it as a part of the packaging. It was this, or writing a lot of upstream C code.
+- name: ezquake-data
+  buildsystem: simple
+  build-commands:
+  - mkdir /app/share  
+  # required for all pre-packaged files
+  - mv ezquake /app/share/ezquake 
+  - ln -s ~/.ezquake/ezquake /app/bin/ezquake
```

Comment:
Does that actually work? `build-commands` happen on build time, so `~` will point to different place depending on user used to run `flatpak-builder`.

Reply:
You actually hit the problem caused by this at https://github.com/flathub/flathub/issues/1175.

I haven't checked myself, but does the binary check it's absolute location or just current working directory?

Reply:
I've changed it using a git-patch. The symlink thing indeed didn't work, but that's one of those things you can't discover when you only compile and run it locally.

On the topic of meta data, I created an issue: https://github.com/ezQuake/ezquake-source/issues/337